source/cloud-security/azure/activity-services/services/storage.rst

```diff
@@ -23,7 +23,12 @@ This section explains how to use the Azure portal to archive the Azure activity
 Configuring the Activity log export
 -----------------------------------
 
-1. To export the logs, search for the **Activity log** service. It can be found by typing "Activity" in the search engine. From there, access the **Audit Logs** section and click on **Export Data Settings**.
+1. To export the logs, search for the **Activity log** service by typing **Activity Log** . It can be found in the **Monitor** service menu or by typing "Activity" in the search engine. From there, click on **Export Data Settings**.
+
+.. thumbnail:: /images/cloud-security/azure/storage-activity-4.png
```

Comment:
This image doesn't show the **Export Data Settings** button mentioned above. It's showing an **Export activity logs** button instead.